extensions/apps/extensions/src/components/pages/info-page/info-page.tsx

```diff
@@ -284,7 +275,7 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
                     <Card onClick={handleDeveloperClick} type="plain">
                       <Stack direction="row" align="center">
                         <ProfileAvatarButton
-                          profileId={appData.author?.akashaProfile?.did?.id}
+                          profileId={appData.author?.id}
```

Comment:
This change is because the author might not have a profile? And the author id is actually the did?

Reply:
yes, aparently the app's author.id is the did